From epasheva@sierrawireless.com  Fri Oct  9 11:01:26 2009
From: Elina Pasheva <epasheva@sierrawireless.com>
Date: Thu, 17 Sep 2009 15:26:20 -0700
Subject: USB: serial: sierra driver version change to 1.3.8
To: "gregkh@suse.de" <gregkh@suse.de>
Cc: <epasheva@sierrawireless.com>, <rfiler@sierrawireless.com>, Oliver Neukum <oliver@neukum.org>
Message-ID: <1253226380.23934.4.camel@Linuxdev3>


Subject: [PATCH 001/001] USB: serial: sierra driver version change to 1.3.8
From: Elina Pasheva <epasheva@sierrawireless.com>

Updated sierra driver version from 1.3.7 to 1.3.8 now that the autosuspend
capabilities were added to the driver.

Signed-off-by: Elina Pasheva <epasheva@sierrawireless.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/sierra.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/serial/sierra.c
+++ b/drivers/usb/serial/sierra.c
@@ -17,7 +17,7 @@
   Whom based his on the Keyspan driver by Hugh Blemings <hugh@blemings.org>
 */
 
-#define DRIVER_VERSION "v.1.3.7"
+#define DRIVER_VERSION "v.1.3.8"
 #define DRIVER_AUTHOR "Kevin Lloyd, Elina Pasheva, Matthew Safar, Rory Filer"
 #define DRIVER_DESC "USB Driver for Sierra Wireless USB modems"
 
